starter_code/relaxr.html

```diff
@@ -0,0 +1,67 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <title>Relaxr</title>
+  </head>
+  <body>
+
+    <div class="section-one">
```

Comment:
I LOVE that you broke each section into it's own `<div>`. I'd probably have the class names be more meaningful, like `header`, `hero`, `footer`, etc instead of `section-one`, `section-two`, etc